### PR TITLE
Pool Group

### PR DESCRIPTION
**Pull request by chaitanyaavi**
Added support to convert F5 priority-group to Avi pool group

### CODE REVIEW

File: python/avi/f5_converter/conversion_util.py [hunk 1]
```diff
@@ -533,6 +533,14 @@ def add_ssl_to_pool(avi_pool_list, pool_ref, pool_ssl_profiles):
                 pool["ssl_key_and_certificate_ref"] = pool_ssl_profiles["cert"]
 
 
+def add_ssl_to_pool_group(avi_config, pool_group_ref, ssl_pool):
+    pool_group = [obj for obj in avi_config['PoolGroup']
```
Comment (grastogi23) [inline python/avi/f5_converter/conversion_util.py:537]: documentation

File: python/avi/f5_converter/conversion_util.py [hunk 2]
```diff
@@ -566,6 +574,19 @@ def update_pool_for_persist(avi_pool_list, pool_ref, persist_profile,
     return pool_updated
 
 
+def update_pool_group_for_persist(avi_config, pool_ref, persist_profile,
+                            hash_profiles, persist_config):
+
```
Comment (grastogi23) [inline python/avi/f5_converter/conversion_util.py:579]: documentation

**Comment by grastogi23** [comment]
can you pl. add documentation and unit tests.

**Comment by chaitanyaavi** [comment]
Added unit tests for Pool group

**Review by grastogi23 — APPROVED** [review]

**Pull request merged**